marti_common_msgs/msg/HealthStatus.msg

```diff
@@ -0,0 +1,4 @@
+Header header
+
+bool   healthy
+string message
```

Comment:
When discussing this with Matt and Marc, one thing we noted is that this isn't message isn't necessarily a health check for an entire node, but just for a component of it; for example, a Microstrain driver could produce independent "imu_health" and "gps_health" topics.  A node's overall health could be different from some subsystem of it.

Path planning node with invalid GPS: Can the path planner detect if the GPS is invalid?  If not, it'd probably report OK, since it would have no way to know otherwise.  If so, can the path planner still plan around obstacles or terrain without GPS?  If it can, then WARN, since it can still produce a path but it may not be optimal (or correct).  If it can't produce a path at all, then ERROR.

On the receiver side, similarly, it'd make sense for the status to be WARN if it quality is bad or intermittent, but ERROR if there are issues that would prevent it from ever being able to produce a fix (such as not having a connected antenna or having zero satellites).